ui/pages/confirmation/confirmation.js

```diff
@@ -362,6 +375,18 @@ export default function ConfirmationPage({
         onCancel={templatedValues.onCancel}
         submitText={templatedValues.submitText}
         cancelText={templatedValues.cancelText}
+        loadingText={templatedValues.loadingText}
+        loading={loading}
+        submitAlerts={submitAlerts.map((alert, idx) => (
+          <Callout
+            key={alert.id}
+            severity={alert.severity}
+            dismiss={() => dismissAlert(alert.id)}
```

Comment:
There appears to be no way to dismiss these alerts. The `dismissAlert` function here is not for the "submit alerts" (which are component state), it's for the "alert state" reducer. When this line is called, it won't work because the reducer isn't tracking this id.